x/gmp/client/cli/tx.go

```diff
@@ -49,12 +51,24 @@ func GetCmdRelay() *cobra.Command {
 			if args[1] == "" {
 				return fmt.Errorf("destination-address cannot be empty")
 			}
+			if args[2] == "" {
+				return fmt.Errorf("command-selector cannot be empty")
+			}
 			if args[3] == "" {
+				return fmt.Errorf("command-params cannot be empty")
+			}
+			if args[4] == "" {
+				return fmt.Errorf("timestamp cannot be empty")
+			}
+			if args[5] == "" {
 				return fmt.Errorf("denoms cannot be empty")
 			}
+			if args[6] == "" {
+				return fmt.Errorf("amount cannot be empty")
+			}
 
-			// Normalize the coin denom
-			coin, err := sdk.ParseCoinNormalized(args[2])
+			// normalize the coin denom
+			coin, err := sdk.ParseCoinNormalized(args[6])
```

Comment:
I think the token amount shouldn't be a required argument in this case since its only used for `HandleGeneralMessageWithToken` while `HandleGeneralMessage` doesn't use it.